src/ontology/components/cellxgene_subset.owl

```diff
@@ -703,2747 +703,2752 @@ Declaration(AnnotationProperty(<http://www.geneontology.org/formats/oboInOwl#inS
+# Class: <http://purl.obolibrary.org/obo/CL_4023050> (L6 intratelencephalic projecting glutamatergic neuron of the primary motor cortex)
 
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CL_4023050> <http://purl.obolibrary.org/obo/cl#cellxgene_subset>)
 
-# Class: <http://purl.obolibrary.org/obo/CL_4023051> (<http://purl.obolibrary.org/obo/CL_4023051>)
+# Class: <http://purl.obolibrary.org/obo/CL_4023051> (vascular leptomeningeal cell)
 
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CL_4023051> <http://purl.obolibrary.org/obo/cl#cellxgene_subset>)
 
-# Class: <http://purl.obolibrary.org/obo/CL_4023054> (<http://purl.obolibrary.org/obo/CL_4023054>)
+# Class: <http://purl.obolibrary.org/obo/CL_4023054> (mesothelial fibroblast)
 
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CL_4023054> <http://purl.obolibrary.org/obo/cl#cellxgene_subset>)
 
-# Class: <http://purl.obolibrary.org/obo/CL_4023056> (<http://purl.obolibrary.org/obo/CL_4023056>)
+# Class: <http://purl.obolibrary.org/obo/CL_4023056> (vascular leptomeningeal cell (Mmus))
 
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CL_4023056> <http://purl.obolibrary.org/obo/cl#cellxgene_subset>)
 
-# Class: <http://purl.obolibrary.org/obo/CL_4023059> (<http://purl.obolibrary.org/obo/CL_4023059>)
+# Class: <http://purl.obolibrary.org/obo/CL_4023059> (differentiation-committed oligodendrocyte precursor)
 
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CL_4023059> <http://purl.obolibrary.org/obo/cl#cellxgene_subset>)
 
-# Class: <http://purl.obolibrary.org/obo/CL_4023065> (<http://purl.obolibrary.org/obo/CL_4023065>)
+# Class: <http://purl.obolibrary.org/obo/CL_4023065> (meis2 expressing cortical GABAergic cell)
 
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CL_4023065> <http://purl.obolibrary.org/obo/cl#cellxgene_subset>)
 
-# Class: <http://purl.obolibrary.org/obo/CL_4023070> (<http://purl.obolibrary.org/obo/CL_4023070>)
+# Class: <http://purl.obolibrary.org/obo/CL_4023070> (obsolete caudal ganglionic eminence derived GABAergic cortical interneuron)
 
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CL_4023070> <http://purl.obolibrary.org/obo/cl#cellxgene_subset>)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/CL_4023070> "caudal ganglionic eminence derived GABAergic interneuron
+and caudal ganglionic eminence derived interneuron are the same cell type. 
+The two must be merged and one needs to be defined as obsolete. 
+We decided to deprecate the entity with the more specific term and redundant qualities 
+in the label. See comments in issue 2357.")
```

Comment:
This subset is auto-generated, so this annotation will be deleted next time. It should move to the cl-edit.owl file.

Reply:
So there is no action to take here, right?